barman/retention_policies.py

```diff
@@ -170,6 +195,15 @@ def _backup_report(self, source):
         # NOTE: reverse key orders (simulate reverse chronology)
         i = 0
         for bid in sorted(backups.keys(), reverse=True):
+            if backups[bid].is_incremental:
+                _logger.debug(
+                    "Ignoring incremental backup %s. The retention status will"
+                    " be propagated from %s if parent status is VALID,"
+                    " OBSOLETE, POTENTIALLY OBSOLETE. If parent has any KEEP*"
+                    " status, incremental backup will be VALID."
+                    % (backups[bid], backups[bid].parent_backup_id)
+                )
```

Comment:
```suggestion
                _logger.debug(
                    "Ignoring incremental backup %s. The retention status will"
                    " be propagated from %s."
                    % (backups[bid], backups[bid].parent_backup_id)
                )
```

Let's use the same log message in both cases (recovery window and redundancy checks)